csrankings-a.csv

```diff
@@ -147,6 +147,7 @@ Abhradeep Guha Thakurta,Univ. of California - Santa Cruz,https://aguhatha.github
 Abhradeep Thakurta,Univ. of California - Santa Cruz,https://aguhatha.github.io,1rV69hMAAAAJ
 Abilio Lucena,UFRJ,http://www.cos.ufrj.br/index.php/pt-BR/pessoas/details/18/2201-abiliolucena,_YS2DcIAAAAJ
 Abir Das,IIT Kharagpur,http://cse.iitkgp.ac.in/~adas,L4yEk2UAAAAJ
+Abir De,IIT Bombay,https://cse.iitb.ac.in/~abir,9ZKKbIAAAAJ

```

Comment:
Web page doesn't load for me; please just replace with the other one mentioned.

Reply:
Thanks! Will do it now.